### ISSUE TITLE
NOTE: Instead of opening issues in github, please consider creating a new topic in https://forum.biobakery.org/. Please read for more details.

### ISSUE THREAD

**Issue by sagun98** [biobakery/kneaddata#9]
[The bioBakery support forum](https://forum.biobakery.org/) provides software support and tutorials for methods for microbial community profiling developed by the Huttenhower lab. Please consider creating a new topic in [the bioBakery support forum](https://forum.biobakery.org/) before opening issues in Github.

**Comment by cjfields** [comment]
Hi @sagun98 , a number of issues have been reported in the forums but have not been addressed or replied to. For example, all three of the following have similar issues.  We are also seeing this problem.  Should we add a ticket here or hold out hope that someone will reply there?

https://forum.biobakery.org/t/paired-end-data-results-in-unpaired-output/928/12
https://forum.biobakery.org/t/paired-end-data-showing-error-on-bypass-trim-option/2759/7
https://forum.biobakery.org/t/kneaddata-a-non-very-explicit-error-message/3334/2

**Comment by cjfields** [comment]
@sagun98 it also looks like the GH actions shut this stub ticket down even though I just commented on it, apologies!

**Comment by ljmciver** [comment]
> Hi @sagun98 , a number of issues have been reported in the forums but have not been addressed or replied to. For example, all three of the following have similar issues. We are also seeing this problem. Should we add a ticket here or hold out hope that someone will reply there?
> 
> https://forum.biobakery.org/t/paired-end-data-results-in-unpaired-output/928/12 https://forum.biobakery.org/t/paired-end-data-showing-error-on-bypass-trim-option/2759/7 https://forum.biobakery.org/t/kneaddata-a-non-very-explicit-error-message/3334/2

Hi @cjfields Thank you for the heads up. I think I got a response to all of them. Please let me know if we are missing any in the future. Feel free to DM me in discourse. 

Sorry kneaddata is not more clear. It actually does not matter what the file names are but the formatting of the read identifiers for older versions. The newer version lets the user specify the R1 and R2 which I think should make it clearer. Feel free to reach out with any additional questions/issues.

Thanks!
Lauren

**Comment by g0ddengr** [comment]
I made an installation of your application today for an HPC user. I wanted to let you know that your pip install recommendation for v0.12.0 doesn't automatically install the required dependencies (trimmomatic and bowtie2) as indicated in your documentation. I was able to install them via conda. However, the "java -Xmx500m -jar /path/to/bin/trimmomatic" command executed by your code expects a jar file without the extension *.jar. So, it throws an error. I had to copy the the trimmomatic.jar file into the executable directory and name it as "trimmomatic" to get your code to work. Just wanted to let you know.
